mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
         - Windows: windows_installation.md
     - Freqtrade Basics: bot-basics.md
     - Configuration: configuration.md
+    - Strategy Quickstart: freqtrade-101.md
```

Comment:
I'm neither convinced that this is the right place (ordering-wise), nor the right name for this.
Having it however simplifies looking at it in a rendered form (`mkdocs serve` -> localhost:8000) - which helps detect rendering issues :D